scripts/run-osx

```diff
@@ -60,13 +60,13 @@ device=$(/Applications/Genymotion\ Shell.app/Contents/MacOS/genyshell -c "device
 open -a /Applications/Genymotion.app/Contents/MacOS/player.app --args --vm-name "${device##*| }"
 fi
 
-# Install deps, prepare for genymotion and figwheel
-lein deps && npm install && ./re-natal use-android-device "${device_type}" && ./re-natal use-figwheel && npm install
+# Install deps
+lein deps && npm install
```

Comment:
Is the lein bit necessary?

Reply:
it's likely that we can remove it. I will check it